detalles.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
Che si en vez de meter un file asi lo metemos al env como variable?
algo onda asi: 

```
FESTIVALS = "bonnaroo, lollapalooza, coachella"
var festivals = process.env.FESTIVALS.split(", ");
```
y ahi recien evaluamos si esta el array o le damos al default

Reply:
El ejemplo lo robe de aca:

https://stackoverflow.com/questions/31552125/defining-an-array-as-an-environment-variable-in-node-js

Reply:
Otra quizas aun mejor

```
Define :

ANY_LIST = ["A", "B", "C"]
Parse :

const LIST = JSON.parse(process.env.ANY_LIST);
Use :

console.log(Array.isArray(LIST)); // true
consloe.log(LIST[2]); // "C"
```

Reply:
Estuve leyendo antes de hacer el json y me pareció que no era lo más prolijo. Meter un array dentro de un string no es lo que mas me gusta y podría traer otros problemas como una descripción de factura que traiga coma.
Dicho lo anterior, respeto que es tu proyecto y es como vos prefieras, puedo meterlo en el .env.